fluentd-image/v1.14/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM alpine:3.14
-LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.14.4"
+LABEL Description="Fluentd docker image" Vendor="Banzai Cloud" Version="1.14.5"
```

Comment:
why did you change this? in logging_types below it's still 1.14.4

Reply:
yes, this need to revert 👍